requirements.txt

```diff
@@ -1,5 +1,5 @@
 coverage == 3.7.1
-django>=1.4,< 1.7
+django>=1.4,< 1.12
```

Comment:
Since it's requirements for tests - let's use specific versions (like we do in all our other clients)

Reply:
And I would surround logical operators with whitespaces - looks better.